pom.xml

```diff
@@ -657,6 +657,116 @@
             </build>
         </profile>
 
+        <profile>
+            <id>payara-micro-remote</id>
+
+            <properties>
+                <!-- default port -->
+                <payara.httpPort>8080</payara.httpPort>
+                <!-- JMS and JAXWS not supported by Micro -->
+                <skipJMS>true</skipJMS>
+                <skipJAXWS>true</skipJAXWS>
+
+                <!-- Being a WebProfile++, Micro does not support ear archives -->
+                <skipEAR>true</skipEAR>
+
+                <!-- Client-cert needs complicated cert setup, which hasn't been done -->
+                <skipServletClientCertificate>true</skipServletClientCertificate>
+            </properties>
+
+            <dependencies>
+
+                <!-- Java EE based client dependencies to contact a server via WebSocket or REST -->
+                <!-- Not needed by the container, but some tests seem to expect JAXRS client implementation to
+                be on the classpath -->
+                <dependency>
+                    <groupId>fish.payara.arquillian</groupId>
+                    <artifactId>payara-client-ee7</artifactId>
+                </dependency>
+
+                <!-- The actual Arquillian connector -->
+                <dependency>
+                    <groupId>fish.payara.arquillian</groupId>
+                    <artifactId>payara-micro-remote</artifactId>
+                    <version>1.0.Beta4-SNAPSHOT</version>
+                    <scope>test</scope>
+                </dependency>
+            </dependencies>
+            <build>
+                <plugins>
+                    <plugin>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <configuration>
+                            <systemPropertyVariables>
+                                <javaEEServer>payara-remote</javaEEServer>
+                                <payara.httpPort>${payara.httpPort}</payara.httpPort>
+                                <use.cnHost>${use.cnHost}</use.cnHost>
+                                <skipEAR>${skipEAR}</skipEAR>
+                            </systemPropertyVariables>
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+
+        <profile>
+            <!-- Helper profile for launching the server. Before the tests run
```

Comment:
Little confused by this. You are using maven to stop start an instance. Isn't that pretty much just managed?

Reply:
Is this by-chance optional? Could I still deploy Payara Micro and deploy the micro-deployer manually? I can see why you've made the profile, but I'm not going to remember those commands :P I personally expect a bit of setup when using remote profiles.

Reply:
Yes, this is optional. You can either remember this one, or launch it e.g. with deployment from maven repo.